### DISCUSSION TITLE
Cannot taxi after landing

### DISCUSSION THREAD

**Discussion by VicDev69** [flybywiresim/aircraft#3872] [Help]
Flying an ILS approach to EGCC, after landing bring aircraft to stop on runway and clean-up (AP off, flaps up and spoiler disarmed. I then try to taxi off runway, but after turning a few degrees, the aircraft returns to runway heading. I am using the latest stable build.
Any ideas or suggestions to fix this problem, please.

**Comment by Paytheon** [comment]
Curious, what was your autobrake setting at landing? Did it shut off?

**Reply by VicDev69** [reply]
Autobrakes set to medium

**Reply by VicDev69** [reply]
... and yes it did shutoff.

**Reply by CptBayer** [reply]
I figured out, that the copilot function (control aircraft) for taxi didn´t work anymore..... I´ll see, if this is fixed after several updates. I am using custom FBW and...


Greets klaus

**Reply by Hansvanwijhe** [reply]
I had the same happening sometimes to me over all versions as well. Hitting "Escape" and  then return into the simulator most of the time solved it for me.

**Reply by VicDev69** [reply]
Further investigations - flight was from runway, aircraft 'ready to fly'. When I release brakes and aircraft starts to roll, steering using Logitech rudder pedals, the aircraft again turns slightly of course and then returns back to the runway heading, it take off do. Circuit and ILS approach, land and come to a stop. Clean up is complete and trying to taxi causes the same problem. I have tried this with the stable version, prior to Update 3, and with the development version earlier today with the me result.
Cheers

**Comment by Annolago** [comment]
If it returns to runway heading then it is acting like Autoland is on. You will not be able to turn without the plane lining up again. 
What you should do is turn off ILS and approach and then both FD's. Remember on both sides..
But I was not under the impression that it was released yet. At least I can't get it to work..

**Reply by Paytheon** [reply]
Anyone can get the Autopilot version right here off github. Many already have it.

**Reply by VicDev69** [reply]
I downloaded the dev version last night, and tried the experiment to taxi 'off-course, before taking off, with the same results - trying to re-capture the runway heading. I had switched off both FDs, no autopilot and no ATHR. This happens on flight started on runway. 
I have tried starting from a gate and  direction control using the rudder pedals work fine.
My objective, initially, was just to do. Quick flight and land, instead I have to due a start from cold and dark which adds considerably to flight time.

**Reply by Hansvanwijhe** [reply]
> Anyone can get the Autopilot version right here off github. Many already have it.

Where can it be downloaded? As far as I know, the CFBW has this option not yet.  Or am I wrong in thinking so?

Thanks,
Hans
